provider/testdata/data/cldr/cldr-bcp47/bcp47/timezone.json

```diff
@@ -0,0 +1,1919 @@
+{
```

Comment:
New JSON files should be added to testdata/Cargo.toml so that they are automatically updated